tests/base/test_maps.py

```diff
@@ -560,7 +560,7 @@ def test_basic(self):
 
         named_model = wires * model
 
-        named_model.sigma == model[: mesh.shape_cells[2]]
+        assert named_model.sigma == model[: mesh.shape_cells[2]]
```

Comment:
Here I assumed that an assertion was missing, is it right?

Reply:
I think that's correct, but I think this line (and the line a little below) need to be changed to `np.testing.assert_equal`(...)